tests/phpunit/tests/functions/TestConfigWpHome.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * Test the _config_wp_home function
+ *
+ * @group Functions
+ * @covers ::_config_wp_home
+ */
+class Test_for_config_wp_home extends WP_UnitTestCase {
```

Comment:
```suggestion
class Tests_Functions_ConfigWpHome extends WP_UnitTestCase {
```
- Test class names should follow the `Tests_Directory_ClassOrFunctionInTitleCase` format. See [Writing PHPUnit Tests - Test Classes](https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#test-classes).